contrib/cirrus/lib.sh

```diff
@@ -157,7 +157,7 @@ setup_rootless() {
     rootless_gid=$((1500 + RANDOM % 5000))
     msg "creating $rootless_uid:$rootless_gid $ROOTLESS_USER user"
     showrun groupadd -g $rootless_gid $ROOTLESS_USER
-    showrun useradd -g $rootless_gid -u $rootless_uid --no-user-group --create-home $ROOTLESS_USER
+    showrun useradd -g $rootless_gid -u $rootless_uid --no-user-group -G systemd-journal --create-home $ROOTLESS_USER
```

Comment:
emember the RHEL thing where rootless has to be in a magic group in order to view journal? That is now the case in the new rawhide.

Reply:
This needs to be investigated by us, journald is our default if no longer longer works by default for a subset of users than we might need to make our default detection logic smarter and auto fallback to file if that is not already the case.

Reply:
How do we handle RHEL?

Reply:
My understanding is that we ship a containers.conf that sets the default.